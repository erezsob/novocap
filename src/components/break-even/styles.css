.container {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    align-items: center;
    height: 20px;
    width: 90%;
    padding-top: 20px;
    color: white;
    font-weight: bold;
    font-size: 20px;
}

.barContainer {
    margin: 20px;
    width: 150px;
    height: 150px;
    position: relative;
    display: flex;
    flex-direction: column;
    justify-content: center;
}

.title {
    display: flex;
    flex-direction: column;
    padding-top: 13px;
}

.date {
    font-size: 11px;
    font-weight: 300;
}

.addCosts {
    color: #fff;
    font-size: 19px;
}