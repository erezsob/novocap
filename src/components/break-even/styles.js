import { blueMain } from '../../common/variables.css';

export default {
  card: {
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    boxShadow: 'none',
    height: 208,
    backgroundColor: blueMain,
    cursor: 'pointer'
  },
  iconButton: {
    color: 'white',
    width: 24
  }
};
