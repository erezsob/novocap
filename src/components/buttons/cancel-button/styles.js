import { pinkyRed } from '../../../common/variables.css';

export default {
  cancel: {
    color: pinkyRed
  }
};
