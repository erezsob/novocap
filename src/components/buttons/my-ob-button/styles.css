@value blueLight, bpXSmall from '../../../common/variables.css';

.button {
    border-radius: 4px;
    background-color: blueLight;
    color: white;
    padding: 8px 0;
    font-size: 12px;
    display: table;
    border: 0;
    outline: 0;
    cursor: pointer;
}

.bigger {
    padding: 10px 0;
    font-size: 16px;
}

.icon,
.text {
    vertical-align: middle;
    text-align: center;
}

.text {
    padding: 0 20px;
    font-weight: lighter;
}

.icon ~ .text {
    display: none;
}

.icon {
    display: table-cell;
    padding: 0 10px;
    position: relative;
    top: 2px;
}

@media (min-width: bpXSmall) and (min-device-width: bpXSmall) {
    .icon ~ .text {
        padding-left: 0;
        display: table-cell;
    }
}
