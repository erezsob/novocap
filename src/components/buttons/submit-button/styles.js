import { greenGrass } from '../../../common/variables.css';

export default {
  submit: {
    color: greenGrass
  }
};
