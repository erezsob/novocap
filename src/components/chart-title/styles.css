.label {
    letter-spacing: 0.05em;
    font-size: 20px;
    border-bottom: 1px solid #efefef;
    padding: 20px;
    margin-bottom: 20px;
    font-weight: bold;
    color: #323435;
}