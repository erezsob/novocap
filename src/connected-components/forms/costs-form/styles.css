@value gray51 from '../../../common/variables.css';

.textField {
    margin: 34px 0 0;
}

.contentWrapper {
    padding: 24px;
}

.contentText {
    color: gray51;
    font-size: 1rem;
    line-height: 1.5em;
}

.buttons {
    margin: 8px 4px;
    display: flex;
    justify-content: flex-end;
}