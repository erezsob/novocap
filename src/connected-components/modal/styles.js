export default {
  paper: {
    minHeight: 280,
    fontSize: 35,
    width: 450
  }
};
