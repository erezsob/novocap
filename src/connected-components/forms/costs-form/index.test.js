import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import { Provider } from 'react-redux';
import { CancelButton, SubmitButton } from '../../../components';
import ConnectedCostsForm, { CostsForm } from './index';
import styles from './styles.css';

const mockStore = configureStore([thunk]);
const initialState = {
  saveCostsFromModal: jest.fn(),
  hideModal: jest.fn()
};
const store = mockStore(initialState);

describe('<CostsForm />', () => {
  let wrapper;
  const props = {
    label: 'LabelText',
    content: 'Check all this content',
    handleSubmit: jest.fn()
  };

  beforeEach(() => {
    wrapper = shallow(<CostsForm {...props} />);
  });

  it('renders', () => {
    expect(wrapper).toHaveLength(1);
  });

  it('renders connected component properly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <ConnectedCostsForm {...props} />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders with a given label', () => {
    expect(wrapper.find(`.${styles.textField}`).props().label).toBe(
      'LabelText'
    );
  });

  it('renders with a given content', () => {
    expect(wrapper.find(`.${styles.contentText}`).props().children).toBe(
      'Check all this content'
    );
  });

  it('renders buttons', () => {
    expect(wrapper.find(`.${styles.buttons}`)).toHaveLength(1);
    expect(wrapper.find(CancelButton)).toHaveLength(1);
    expect(wrapper.find(SubmitButton)).toHaveLength(1);
  });
});
