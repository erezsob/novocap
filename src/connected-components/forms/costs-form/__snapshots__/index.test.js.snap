// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CostsForm /> renders connected component properly 1`] = `
<form
  onSubmit={undefined}
>
  <div
    className="contentWrapper"
  >
    <div
      className="contentText"
    >
      Check all this content
    </div>
    <div>
      <div
        aria-describedby={undefined}
        className="MuiFormControl-root-1 MuiFormControl-marginDense-3 MuiFormControl-fullWidth-4 textField"
        onBlur={[Function]}
        onDragStart={[Function]}
        onDrop={[Function]}
        onFocus={[Function]}
      >
        <label
          className="MuiFormLabel-root-11 MuiInputLabel-root-5 MuiInputLabel-formControl-6 MuiInputLabel-animated-9 MuiInputLabel-marginDense-7"
          data-shrink={false}
          htmlFor={undefined}
        >
          LabelText
        </label>
        <div
          className="MuiInput-root-17 MuiInput-fullWidth-24 MuiInput-formControl-18 MuiInput-underline-21"
        >
          <input
            aria-invalid={false}
            aria-required={false}
            autoComplete={undefined}
            autoFocus={undefined}
            className="MuiInput-input-25 MuiInput-inputType-29 MuiInput-inputMarginDense-26"
            defaultValue={undefined}
            disabled={false}
            id={undefined}
            name="costs"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            onKeyDown={undefined}
            onKeyUp={undefined}
            placeholder={undefined}
            readOnly={undefined}
            required={undefined}
            rows={undefined}
            type="number"
            value=""
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="buttons"
  >
    <button
      className="MuiButtonBase-root-47 MuiButton-root-32 CancelButton-cancel-31"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="button"
    >
      <span
        className="MuiButton-label-33"
      >
        Cancel
      </span>
      <span
        className="MuiTouchRipple-root-50"
      />
    </button>
    <button
      className="MuiButtonBase-root-47 MuiButton-root-32 SubmitButton-submit-57"
      disabled={false}
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="submit"
    >
      <span
        className="MuiButton-label-33"
      >
        Save
      </span>
      <span
        className="MuiTouchRipple-root-50"
      />
    </button>
  </div>
</form>
`;
